package.json

```diff
@@ -12,19 +12,13 @@
         "prettier": "prettier \"**/**.{ts,tsx,js,jsx,json,css,scss,html}\" --write",
         "prettier:check": "prettier \"**/**.{ts,tsx,js,jsx,json,css,scss,html}\" --check",
         "rnlink": "npx react-native link",
-        "start:blank-ios": "cd demo && yarn start:blank-ios",
-        "start:blank-android": "cd demo && yarn start:blank-android",
         "start:blank-typescript-ios": "cd demo && yarn start:blank-typescript-ios",
         "start:blank-typescript-android": "cd demo && yarn start:blank-typescript-android",
-        "start:routing-ios": "cd demo && yarn start:routing-ios",
-        "start:routing-android": "cd demo && yarn start:routing-android",
         "start:routing-typescript-ios": "cd demo && yarn start:routing-typescript-ios",
         "start:routing-typescript-android": "cd demo && yarn start:routing-typescript-android",
-        "start:authentication-ios": "cd demo && yarn start:authentication-ios",
-        "start:authentication-android": "cd demo && yarn start:authentication-android",
         "start:authentication-typescript-ios": "cd demo && yarn start:authentication-typescript-ios",
         "start:authentication-typescript-android": "cd demo && yarn start:authentication-typescript-android",
-        "precommit": "cd demo && bash ./scripts/buildBlank.sh && cd .. && yarn lint && yarn prettier && yarn generate:licenses",
+        "precommit": "cd demo && bash ./scripts/buildBlankTypescript.sh && cd .. && yarn lint && yarn prettier && yarn generate:licenses",
```

Comment:
Where is this file '.scripts/buildBlankTypescript.sh' ?

Reply:
It's in the demo project. I'm not sure the demo project has been maintained though.